cmd/flowrunner/testdata/default.json

```diff
@@ -53,13 +53,31 @@
     },
     {
         "type": "channel",
-        "url": "http://testserver/assets/channel/57f1078f-88aa-46f4-a59a-948a5739c03d",
-        "content": {
-            "uuid": "57f1078f-88aa-46f4-a59a-948a5739c03d",
-            "name": "Twilio Channel",
-            "address": "+12345671111",
-            "type": "T"
-        }
+        "url": "http://testserver/assets/channel",
```

Comment:
need some extra channels here because we only create messages when have a channel to send it